src/Common/Resources.cs

```diff
@@ -16,14 +16,15 @@ public class Resources
         public const string DEPLOYMENT_OUTPUTDIR_HELP = "Directory to place any artifacts generated by the deployment";
         public const string DEPLOYMENT_INTERACTIVE_HELP = "Flag that controls if the user should be prompted to go to the portal to initialize the installation manually";
         public const string DEPLOYMENT_TERMINATE_OTHER_VERSIONS_HELP = "Flag that controls if all the other versions of the Customer Environment should be terminated";
-        public const string DEPLOYMENT_TIMEOUT_MINUTES = "Number of minutes that are allowed to wait for the deployment to succeed. The default is 60 minutes.";
+        public const string DEPLOYMENT_TIMEOUT_MINUTES = "Timeout, in minutes, that are allowed to wait for the deployment to succeed. The default is 360 minutes (6 hours).";
+        public const string DEPLOYMENT_TIMEOUT_MINUTES_TO_GET_SOME_MB_MESSAGE = "Timeout, in minutes, that SDK client will wait to receive any message from portal by Message Bus. The default is 30 minutes.";
```

Comment:
"Timeout, in minutes, that the SDK client waits to receive any message from the portal via Message Bus. The default is 30 minutes."